nntrainer/graph/network_graph.cpp

```diff
@@ -426,14 +436,51 @@ void NetworkGraph::backwarding(
   if (clip_weights.empty())
     return;
 
+  // update loss scale
```

Comment:
It might be worth mentioning that mixed-precision training needs to enable clip_weights. Well, this is not exactly the right comment but it is necessary to compute "apply gradient" after checking if the weight is NaN or Inf.